pkg/utils/cluster_info.go

```diff
@@ -0,0 +1,34 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	clusterIDConfMap = "cluster-id"
+	// Todo: Is there a place where i can get this?
+	configMapNamespace = "liqo"
+)
+
+func GetClusterID(c client.Client) (string, error) {
+
+	configMaps := &corev1.ConfigMapList{}
+	// possibility to specify "Liqo" namespace if it is possible to get it somewhere
+	if err := c.List(context.TODO(), configMaps, client.InNamespace(configMapNamespace)); err != nil {
```

Comment:
Why are you listing all configmaps and not getting directly the correct one (by name or by label)? This seems very fragile to me

Reply:
I have understood that listing resources was a best practise in order to avoid transient condition, just in case someone replicates the resources or a new resource is spawn due to transient errors. During my development I see only one ConfigMap in the liqo namespace, my fault, thank you.